packages/pangea-node-sdk/src/types.ts

```diff
@@ -2509,3 +2515,75 @@ export namespace AuthZ {
     subjects: Subject[];
   }
 }
+
+export namespace Sanitize {
+  export interface SanitizeFile {
+    scan_provider?: string;
+  }
+
+  export interface SanitizeContent {
```

Comment:
Should we add docs now that API is almost stable?

Reply:
Done.